### DISCUSSION TITLE
2023-09-27 Weekly Meeting 126 Notes/Agenda

### DISCUSSION THREAD

**Discussion by indiebio** [omigroup/omigroup#419] [Weekly Meetings]
You can add it to your calendar [here](https://calendar.google.com/calendar/u/1?cid=Y18wZHB1Z2Y5ZjgzZXE0cWVrbWI2b21xYmptZ0Bncm91cC5jYWxlbmRhci5nb29nbGUuY29t).

We'll be meeting again in the omi-weekly-meeting channel of the [AngellXR/OMI Discord](https://discord.gg/NJtT9grz5E).
We regularly use the # omi-meeting-chat channel for text chat.

Who we are and why we're here:
The Open Metaverse Interoperability Group (OMI) is focused on bridging virtual worlds by designing and promoting protocols for identity, social graphs, inventory, and more. Our members include businesses and individuals working towards this common goal.

If you’re an observer or here for the first time, this is a meeting where people who work on the OMI group initiatives discuss updates and contribution opportunities for current projects and goals. This is your chance to jump in and help!

We welcome you to introduce yourself in the 🔈 omi-weekly-meeting chat as we dive into the agenda items. There will be an open floor in the last 15 minutes of the meeting (agenda permitting) where we can discuss any items you feel are relevant to the current or near future OMI goals.

Agenda:

1. Metatraversal: https://discord.com/channels/770382203782692945/851927319173791806/1154294930563399721
2. 30 September (this weekend) - progress and more planning: https://omi-physics.uwu.ai/; https://discord.com/channels/770382203782692945/1149030210503716944
3. Venue for demo day etc? opinions on Stardust XR? Relevant? -> Hyperfy. Stardust is a different thing, good to discuss in a demo day :)
4. election??
5. help needed for Open Collective - mr Metaverse
6. Common (?) access to e.g. calendar, website updates...

**Comment by mrmetaverse** [comment]
# It's an exciting time to join OMI! 
- we are in a creative phase 
- we are building together 
- we are celebrating fresh voices and perspectives

**Reply by indiebio** [reply]
Low hype is a great time to build good quality - focus on depth and quality, not hype

**Reply by indiebio** [reply]
It's also about implementers and contributors. It's still hard to do so much work with so few people.
The vision will endure, as the cycle goes up and down people go in and out.
It's also about managing expectations. There's a pet theory that of every 100 people who express some interest, 10 will follow up and maybe 1 out of 100 will have staying power.

**Reply by mrmetaverse** [reply]
One observation shared is that many don't feel they are worthy or the right people to be talking about standards. They get excited about the idea of being able to make assets in blender and and load into and across multiple game engines yet still feel "what do you want us to do about it" "what is our input worth, don't you need to discuss this with Khronos"

**Reply by mrmetaverse** [reply]
Others don't want to "muddy the water". Aaron shared that an ocean of muddy water is better than the water all drying up. 

I think this drives a CTA for agency. We may need more visible agency to reinforce that we can make a difference and we do have seat(s) at the table(s).

**Reply by indiebio** [reply]
On gTF: 
This is a BabalyonJS session they had last week discussing GLTF extensions for animation and what they are doing in that engine ........
https://youtu.be/2q0xGIJp06o?si=mSLcj5Tdjcjjqpj_&t=2672

**Reply by indiebio** [reply]
a discussion about a few drops of a crystal clear spring vs a muddy ocean... We need a muddy ocean to filter from. The agreed issues on github is the crystal clear spring. So where does the muddy ocean come from? People adding their passions, what they are interested in. Then the BabylonJS link was shared as example. Need people to bring their half baked ideas, to see where there is overlap, and we can help each other. That is the muddy ocean, and anyone and everyone can contribute to that.

**Comment by indiebio** [comment]
Have more watchparties, like a hyperfy world that is a movie theatre, quick, doesn't need much planning. Has the added benefit over e.g. just watching a youtube because you can chat to people sitting in the theatre

**Reply by mrmetaverse** [reply]
https://hyperfy.io/watchparty/ Meta Connect event today

**Reply by indiebio** [reply]
Once a week, like indie movies and stuff. Make a list and just have one showing every week. The biggest issue is what time zone, something that works OK for both US and Europe.

**Comment by indiebio** [comment]
TIL: OMI has a youtube channel! https://www.youtube.com/@Open_Metaverse

**Reply by mrmetaverse** [reply]
- we want to share your content! If you/company has youtube content about the open metaverse please share your links in omi-media to get added to our [playllst](https://www.youtube.com/@Open_Metaverse/playlists).

**Reply by mrmetaverse** [reply]
- right now the process is simple, just ping @mrmetaverse in omi-media-links and they will coordinate getting your blog routed

**Comment by mrmetaverse** [comment]
We also have a Blog where we can cross post content. 

- make a process for submitting blogs that folks want to cross-post from OMI acct. OMI acct publishes an abstract of the original and a link.

**Reply by mrmetaverse** [reply]
- right now the process is simple, just ping @mrmetaverse and they will coordinate getting your blog routed

**Reply by indiebio** [reply]
Update when working groups meet, update the event reminders in Discord. Add that page about when working groups meet somewhere obvious like the website

**Comment by mrmetaverse** [comment]
# OMI Demo Day - Physics for the Open Metaverse
>## This Saturday September 30, 2023 @ 2PM EDT (18:00 UTC)  

https://omi-physics.uwu.ai/

[![image](https://github.com/omigroup/omigroup/assets/63426722/2a4365e4-a74d-46cd-8196-6370a140f252)](https://omi-physics.uwu.ai/)

**Reply by mrmetaverse** [reply]
- [ ] update due to spelling mistake
- [ ] blast out on socials 
- [ ] make a YT announcement (short)

**Comment by mrmetaverse** [comment]
# Election (?) 

Why have one? 
- we are due for one soon 
- its something we committed to doing when we started having elections 
- Open Collective is ready to help us make the switch to 501c3 sponsorship, but they need point people. 
- there are roles and responsibilities. 
- W3C needs to be updated, we want to bridge the W3C list with our Discord community, spin up a proper newsletter to update the mass collective, and engage more effectively.

**Comment by mrmetaverse** [comment]
# MetaTraversal Event tomorrow 
- sign up here https://www.linkedin.com/events/metatr-versal-portalcrawliii7107931217995407361/about/
